docs/preview/02-Features/06-general-messaging.md

```diff
@@ -9,107 +9,6 @@ Both and future systems also support some general functionality that will be exp
-        }
-    }
-}
-```
-
-The message pump will by default act in the following pattern:
-* Circuit breaker calls `Pause`
-  * Message pump stops processing messages for a period of time (circuit is OPEN).
-* Message pump tries processing a single message (circuit is HALF-OPEN).
-  * Dependency still unhealthy? => circuit breaker pauses again (circuit is OPEN)
-  * Dependency healthy? => circuit breaker resumes, message pump starts receiving message in full again (circuit is CLOSED).
-
-Both the recovery period after the circuit is open and the interval between messages when the circuit is half-open is configurable when calling the circuit breaker. These time periods are related to your dependent system and could change by the type of transient connection failure.
-
-```csharp
- public override async Task ProcessMessageAsync(
-        Order message,
-        AzureServiceBusMessageContext messageContext,
-        MessagePumpCircuitBreakerOptions options,
-        ...)
-{
-    // Sets the time period the circuit breaker should wait before retrying to receive messages.
-    // A.k.a. the time period the circuit is closed (default: 30 seconds).
-    options.MessageRecoveryPeriod = TimeSpan.FromSeconds(15);
- 
-    // Sets the time period the circuit breaker should wait between each message after the circuit was closed, during recovery.
-    // A.k.a. the time interval to receive messages during which the circuit is half-open (default: 10 seconds).
-    options.MessageIntervalDuringRecovery = TimeSpan.FromSeconds(1.5);
-}
-```
-
-#### 🔔 Get notified on a circuit breaker state transition
-Transitions from circuit breaker states happens internally and automatically. The library supports a notification system that lets you register event handlers that gets called upon a state transition.
-
-After the message pump and/or message handlers are registered, you can add one or more handlers linked to specific transitions to the previously registered pump.
-
-```csharp
-using Arcus.Messaging.Pumps.Abstractions;
-
-services.AddServiceBusMessagePump(...)
-        .WithCircuitBreakerStateChangedEventHandler<MyFirstCircuitBreakerEventHandler>()
-        .WithCircuitBreakerStateChangedEventHandler<MySecondCircuitBreakerEventHandler>();
-```
-
-The instances should implement the `ICircuitBreakerEventHandler`, which allows you to access the new state for the pump.
-
-```csharp
-public class MyFirstCircuitBreakerEventHandler : ICircuitBreakerEventHandler
-{
-    public Task OnTransitionAsync(MessagePumpCircuitStateChang change)
-    {
-        // The job ID of the message pump that was transitioned.
-        string jobId = change.JobId;
-
-        // The circuit breaker state transitions.
-        MessagePumpCircuitState oldState = change.OldState;
-        MessagePumpCircuitState newState = change.NewState;
-
-        // Process the state change event...
-    }
-}
-```
-
 ### Pause message processing for a fixed period of time 
```

Comment:
Depends on what we decide with the `IMessagePumpLifetime` functionality, otherwise, we can remove this page in its entirety.

Reply:
Removed the file here: https://github.com/arcus-azure/arcus.messaging/pull/546/commits/81b1ce91d81abbe5fb3af7c284ac6b724c255051